Description: Use jackson-bom (deprectated) instead of jackson-parent
 jackson-module-mrbean (2.6.7-1) UNRELEASED; urgency=medium
 .
   * Used jackson-module-jaxb-annotations as a starting point for jackson-module-mrbean
Author: Ignacio Losiggio <iglosiggio@gmail.com>

---
Last-Update: 2017-12-28

--- jackson-module-mrbean-2.6.7.orig/pom.xml
+++ jackson-module-mrbean-2.6.7/pom.xml
@@ -3,8 +3,8 @@
   <modelVersion>4.0.0</modelVersion> 
   <parent>
     <groupId>com.fasterxml.jackson</groupId>
-    <artifactId>jackson-parent</artifactId>
-    <version>2.6.2</version>
+    <artifactId>jackson-bom</artifactId>
+    <version>debian</version>
   </parent>
   <groupId>com.fasterxml.jackson.module</groupId>
   <artifactId>jackson-module-mrbean</artifactId>
